core/client/fs/src/test/java/alluxio/client/block/stream/GrpcDataWriterTest.java

```diff
@@ -289,10 +291,11 @@ private long verifyWriteRequests(final BlockWorkerClient client, final long chec
    */
   private void validateWriteRequest(WriteRequest request, long offset) {
     if (request.hasCommand()) {
-      Assert.assertEquals(BLOCK_ID, request.getCommand().getId());
-      Assert.assertEquals(offset, request.getCommand().getOffset());
+      assertEquals(BLOCK_ID, request.getCommand().getId());
+      assertEquals(offset, request.getCommand().getOffset());
     } else {
-      Assert.assertTrue(request.hasChunk());
+      assertTrue(request.hasChunk());
     }
   }
 }
+
```

Comment:
remove extra newline

Reply:
Thanks for your advice, I have resolved these problem and hope it meets your request.